ngmix/priors.py

```diff
@@ -3017,36 +3029,36 @@ def get_lnprob_array2d(self, eta1, eta2):
     '''
 
 
-class Normal(object):
+class Normal(PriorBase):
     """
     This class provides an interface consistent with LogNormal
 
     C class defined get_lnprob_scalar(x) and get_prob_scalar(x)
     """
```

Comment:
Is there any C left here?

Reply:
nope

Reply:
Cool we should update the doc strings when we have time.

Reply:
fixed now